lib/dry/logic/build.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require_relative "build/base"
+require_relative "build/predicate"
+require_relative "build/operation"
+
+module Dry
+  module Logic
+    module Build
+      def call(&block)
+        Operation.call(&block)
+      rescue NoMethodError
```

Comment:
This messes up the stack trace/error message. Right now `NoMethodError` on `Predicate` is thrown for all non-existing calls, no matter if the caller intended to invoke a predicate or an operation. The error message should say that method invoked was missing on both `Operation` and `Predicate`, not just `Predicate`.

Reply:
Fixed in b16015c.